server/models.py

```diff
@@ -11,57 +12,83 @@
 
 from constants import max_size
 
-#=====================================================
-# Superclasses
-#=====================================================
 
-
-class Model(Document):
```

Comment:
any reason we tossed model?
even if we don't use it we're still probably gonna need the save() function to be overriden in each class so we can update the 'updated' field

Reply:
Yea, this structure was apart the reason why it wasn't working how we wanted it. Mongo was creating an index on email preventing it from appearing again in any of the documents. Since `User` and `Map` were subclasses of `Model`  their unique constraints were applied to `Model`.  I'll add the save override back in for both `GameMap` and `User` though.